layouts/docs/search.html

```diff
@@ -1,30 +0,0 @@
-{{ define "main" }}
-    <div class="container mx-auto px-4 py-8">
-        <div class="md:flex">
-            <div class="w-full">
-
-                <div class="md:flex items-center mb-4">
-                    <div class="mb-4 md:ml-auto md:order-last">
-                        {{ partial "docs/search.html" . }}
-                    </div>
-                    <h1 class="no-anchor mt-0">{{ .Title }}</h1>
-                </div>
-
-                <div class="my-4 text-gray-500" id="search-spinner">
-                    <i class="fas fa-sync-alt spinner text-xl"></i>
-                </div>
-
-                <div class="md:flex items-start" id="search-results"></div>
-            </div>
-        </div>
-    </div>
-
-    <!--
-        Minify and fingerprint script. Generated file will contain content hash in
-        filename, so no need to add query param to cache bust. Also note that because this
-        script references functions defined in the main bundle, we defer loading it until
-        that script's been evaluated.
-    -->
-    {{ $js := resources.Get "js/search.js" | resources.Minify | resources.Fingerprint }}
```

Comment:
`assets/js/search.js` and `static/js/search-worker.js` can be deleted.

If I'm remembering correctly, `assets/sass/_search.scss` can also be deleted, along with the import for it in `assets/sass/styles.scss`:

https://github.com/pulumi/docs/blob/26a033fb30a38ebc19eeac0b395ad381c597b713/assets/sass/styles.scss#L30